rfc/0032-field-level-encryption.md

```diff
@@ -728,16 +729,24 @@ There are a few items out of scope at this time for v1.0, however they very much
   * Algorithm upgrading - for example going from AES-256 to AES-512
   * Support for KMIP based key stores
 
+
+## Changelog
+
+YYYY-MM-DD Initial draft publication
```

Comment:
YYYY-MM-DD should be specified and can be determined vaguely based on when this was first contributed to Git.